lib/uvm_agents/uvma_axi5/src/uvma_axi_pkg.flist

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 Thales DIS SAS
+//
+// Licensed under the Solderpad Hardware Licence, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// SPDX-License-Identifier: Apache-2.0 WITH SHL-2.0
+// You may obtain a copy of the License at https://solderpad.org/licenses/
+//
+// Original Author: Alae Eddine EZ ZEJJARI (alae-eddine.ez-zejjari@external.thalesgroup.com) – sub-contractor MU-Electronics for Thales group
+// Co-Author: Abdelaali Khardazi
+
+// Directories
++incdir+${DV_UVMA_AXI_PATH}
```

Comment:
These variables conflict with variables of the same name in `lib/uvm_agents/uvma_axi`.    We either need to rename these variables (e.g. `DV_UVMA_AXI5`) or delete the AXI Agent in `lib/uvm_agents/uvma_axi`.